### ISSUE TITLE
Horizontal scroll bar on iOS 

### ISSUE THREAD

**Issue by KevinColemanInc** [amitmerchant1990/reverie#1]
On my iPhone X, I get a horizontal scroll bar with safari.

**Comment by Fastidious** [comment]
Yes, it doesn’t look too good on mobile. Have to pinch screen to make it fit too.

**Comment by josemariogutierrez** [comment]
Hi! Can i work on this issue?

**Comment by Fastidious** [comment]
@josemariogutierrez I don't see why not. Send the PR to @amitmerchant1990, I am sure he will not mind.

**Comment by josemariogutierrez** [comment]
@Fastidious Cool, thanks!

**Comment by amitmerchant1990** [comment]
@josemariogutierrez You can go ahead. 👍
